django_app/redbox_app/redbox_core/models.py

```diff
@@ -27,13 +27,37 @@ class Meta:
         ordering = ["created_at"]
 
 
+class UserGrade(UUIDPrimaryKeyBase):
```

Comment:
can this not just be an emun?

Reply:
I didn't want to hard-code it. I agreed with Sam that he'd be able to add, rename, and if necessary delete grades via the admin interface.